modules/microshift-viewing-audit-logs.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+//microshift_troubleshooting/microshift-audit-logs.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-security-context-constraints-alert-eval_{context}"]
+= Identifying pod security violations through audit logs
+
+The PodSecurityViolation alert does not provide details on which workloads are causing pod security violations. You can identify the affected workloads by viewing the server audit logs. The following procedure shows you how to access the audit logs and parse them to find security violations.
+
+.Prerequisites
+
+* You have installed `jq`.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. To view the audit logs, run the following command: 
++
+[source,terminal]
+----
+$ oc adm node-logs –role=master –path=kube-apiserver/----
```

Comment:
```suggestion
$ NODE_NAME=$(oc get node -ojsonpath='{.items[0].metadata.name}')
$ oc adm node-logs "${NODE_NAME}" --path=kube-apiserver/ 
```

Reply:
Test driving the commands, I'm getting the node-name as a prefix json-formatted log.  `jq` can't handle that and we can disable it by providing the node-name to fetch the logs from.